README.md

```diff
@@ -152,6 +152,10 @@ The full API Reference is available here.
 * [`GetBucketNotification`](https://docs.minio.io/docs/golang-client-api-reference#GetBucketNotification)
 * [`DeleteBucketNotification`](https://docs.minio.io/docs/golang-client-api-reference#DeleteBucketNotification)
 
+### API Reference : Bucket notification Operations (Minio Extensions)
```

Comment:
This should not be a separate sub-category. This text (Minio Extension) can be added to the side.